templates/app/home-page.phtml

```diff
@@ -96,3 +96,24 @@
         </div>
     </div>
 </div>
+
+<div class="container">
```

Comment:
@froschdesign 

> I think the donation section on the homepage can also be updated, because support for the project is now more than the possibility of a donation.

Good call, I've added a new section below it, as I think keeping the existing section as-is is still valuable.